apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -0,0 +1,220 @@
+<script>
+  import PageSignupCTA from '$lib/PageSignupCTA/index.svelte';
+  import PageHeader from '$lib/PageHeader/PageHeader.svelte';
+
+  const benefitDetails = [
+    {
+      img: 'ai.svg',
+      title: 'AI-Powered Learning',
+      point_1: 'Create engaging courses with AI-assisted content generation.',
+      point_2: 'Automate grading and feedback processes, saving time and effort.',
+      point_3: 'Personalize learning experiences based on individual student needs.'
```

Comment:
Instead of doing this, please use an array rather than naming keys with incrementing orders.

Why?
1. This creates redundant code, which means that to add another point to each object you have to first add another key like this.
2. When you need this value you have to specifically mention their keys before you can use their values
3. Can lead to bugs - assuming I mistakenly say `Point_3` instead of `point_3` in  any of the objects, and you are specifically looking for `point_3` the code can break.


Instead you should do this:

```
{
  points: [
    "Create engaging courses with AI-assisted content generation.",
    "Automate grading and feedback processes, saving time and effort.",
    "Personalize learning experiences based on individual student needs."
  ]
}
```

This way in the html you can do something like:

```
{#each points as point}
  <div class="flex flex-row justify-start items-start gap-x-3">
    <img src="/bootcamps/check.svg" alt="check mark" />
    <p class="font-[#656565] font-light">
      {point}
    </p>
  </div>
{/each}
```